.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,14 +15,15 @@ labels: bug
 
 ## How to reproduce
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
 
 ## Environment
 
-- PostHog cloud or self-managed?
-- PostHog version/commit
+- PostHog Cloud or self-hosted?
```

Comment:
Nitpick: inconsistent use of capitalization

Reply:
Hm, this was intentional, as PostHog Cloud is a product (specifically SaaS) name, while "PostHog Self-Hosted" is not a thing, we just call it "self-hosted PostHog", where "self-hosted" is just an adjective. I think naming is something we haven't _100%_ figured out yet.